data/SMURFCAT/data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Real Smurf Cat",
+  "symbol": "шайлушай",
```

Comment:
Changing it here instead of doing the override in the ethereum token section doesn't fix the issue, since now the symbol in the L2 Base contract doesn't match.

Reply:
Suggest changing this back and adding the override in the L1 ethereum section.  

See for example: https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/11f5bf10eff223d962b47b6aa22f633726ed2cab/data/LIKE/data.json#L12

Reply:
Okay I see! Just did it!

Thank you